app.component.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+//1.
+
+const bookIds = [
+  'oliver_twist',
+  'hamlet',
+  'antony_and_cleopatra',
+  'macbeth',
+  'waiting_for_godot',
+  'arms_and_the_man',
+  'merchant_of_venice',
+  'crime_and_punishment',
+  'romeo_and_juliet',
+  'a_tale_of_two_cities',
+];
+//console.log(bookIds);
+
+//3.
+function books(bookIds) {
```

Comment:
It's better to give a function a more 'active' name, that includes a verb. This is because a function is a tool you're using to get a certain job done. In this case, you could call it "displayBookList" or "renderBookTitles".